vitrina/orgs/models.py

```diff
@@ -82,19 +82,27 @@ def get_absolute_url(self):
 
 
 class Representative(models.Model):
+    COORDINATOR = 'coordinator'
+    MANAGER = 'manager'
+    ROLES = {
+        (COORDINATOR, _("Koordinatorius")),
+        (MANAGER, _("Tvarkytojas"))
+    }
+
     created = models.DateTimeField(blank=True, null=True, auto_now_add=True)
     modified = models.DateTimeField(blank=True, null=True, auto_now=True)
-    version = models.IntegerField()
+    version = models.IntegerField(default=1)
     email = models.CharField(max_length=255, blank=True, null=True)
     first_name = models.CharField(max_length=255, blank=True, null=True)
     last_name = models.CharField(max_length=255, blank=True, null=True)
     organization = models.ForeignKey(Organization, models.DO_NOTHING, blank=True, null=True)
     phone = models.CharField(max_length=255, blank=True, null=True)
     deleted = models.BooleanField(blank=True, null=True)
     deleted_on = models.DateTimeField(blank=True, null=True)
+    role = models.CharField(choices=ROLES, max_length=255, blank=True, null=True)
+    user = models.ForeignKey("vitrina_users.User", models.SET_NULL, blank=True, null=True)
```

Comment:
Do we really add unique to model fields? If same user can be added to multiple organizations, this would not be correct. Maybe we should add validation to forms to not allow same user or email to be added to organization or add unique together for user, organization and email?

Reply:
Should be unique together with organization: https://docs.djangoproject.com/en/3.2/ref/models/options/#unique-together